modules/git/repo_commit.go

```diff
@@ -21,6 +21,11 @@ import (
 func (repo *Repository) GetRefCommitID(name string) (string, error) {
 	ref, err := repo.gogitRepo.Reference(plumbing.ReferenceName(name), true)
 	if err != nil {
+		if err == plumbing.ErrReferenceNotFound {
```

Comment:
Are there any places to check if the error is `plumbing.ErrReferenceNotFound`?

Reply:
Only one other place that I have changed to use the git error.